File: src/gameEntities/modifiers/AttackSpeed1.java
package gameEntities.modifiers;

import dynamicBody.character.player.Stats;
import worldModel.utilities.Pair;
import worldModel.utilities.enums.Entities;

public class AttackSpeed1 extends ModifiersImpl{

	public AttackSpeed1(Pair<Integer, Integer> coord) {
		super(coord, Stats.PROJ_SPEED, 15, "ROF Upgrade", Entities.ATTACKSPEED1);
	}
}

File: Pirazzoli_Federico/GameEntity/AttackSpeed1.cs
using System;

using design.utilities;
using design.enums;


namespace gameEntities.modifiers
{
    public class AttackSpeed1 : ModifiersImpl
    {
        public AttackSpeed1(Pair<int, int> coord) 
            : base(coord, Stats.PROJ_SPEED, 500, "ROF Upgrade", Entities.ATTACKSPEED1)
        {

        }

    }
}
